shell/browser/electron_pdf_web_contents_helper_client.cc

```diff
@@ -9,6 +9,11 @@ ElectronPDFWebContentsHelperClient::ElectronPDFWebContentsHelperClient() =
 ElectronPDFWebContentsHelperClient::~ElectronPDFWebContentsHelperClient() =
     default;
 
+content::RenderFrameHost* ElectronPDFWebContentsHelperClient::FindPdfFrame(
+    content::WebContents* contents) {
+  return nullptr;
```

Comment:
Given the changes made in https://chromium-review.googlesource.com/c/chromium/src/+/3399305, I think this will cause the following code in pdf_web_contents_helper.cc to fail:
```
  pdf_rwh_ = client_->FindPdfFrame(&GetWebContents())->GetRenderWidgetHost();
  pdf_rwh_->AddObserver(this);
```

Reply:
@jkleinsc it looks like 31d1389ffb8ee37b2bec7764f8e89498995b8212 resolves this comment